src/pure/target.mli

```diff
@@ -101,12 +100,12 @@ module Condition : sig
   (**
     An execution anti-condition; the condition defines when a target is
     ready and therefore should be run if the condition is {emph not} met: {ul
-    {li with [`Never] the target always runs (because never “ready”),}
-    {li with [`Satisfied] the target never runs (a bit useless),}
+    {li with [`Never] the target always runs and then fails
+                      (because never “ready”),}
     {li with [`Volume_exists v] the target runs if the volume does not exist
     ([make]-like behavior).}
     {li with [`Volume_size_bigger_than (v, sz)] Ketrew will get the total size
-    of the volume (in bytes) and check that it is bigger.}
+    of the volume (in bytes) and check that it is {b bigger-or-equal}.}
```

Comment:
This is a bit confusingly named; could we use something like "Volume_size_ge_than"?